content/v2/openapi.yml

```diff
@@ -1564,6 +1564,32 @@ paths:
               schema:
                 $ref: '#/components/schemas/Error'
       summary: Check zone distribution
+  '/{account}/zones/{zone}/ns_records':
+    put:
+      description: Updates the zone's NS records
+      parameters:
+        - $ref: '#/components/parameters/Account'
+        - $ref: '#/components/parameters/Zone'
+        - $ref: '#/components/parameters/NsSetIds'
```

Comment:
Should `NsSetIds` be part of a `requestBody` instead?
How will `NsNames` be used, given that we have `NsSetIds` ?